### DISCUSSION TITLE
We are unable to complete your signup.

### DISCUSSION THREAD

**Discussion by aserrajr** [vercel/community#4360] [Account Issue]
### Question

Hello. 

I am trying to complete my sign-up but it says I cannot complete it. 

Can I have some instructions on how to proceed here?

**Comment by amyegan** [comment]
Sorry you're having trouble signing up for an account. We can't safely discuss account information in public because of security concerns. You may have received an email with more details about the error and instructions on next steps. Please check your spam or junk folder in case it was filtered out of your main inbox.

For more info, please check this guide: [Why am I unable to login or signup to the Vercel platform?](https://vercel.com/guides/why-can-i-not-signup)

**Comment by aserrajr** [comment]
@amyegan I did not get any email or instructions on how to proceed. I have sent emails to your support and so far no answer. Actually, you a the first person who answered my questions about this issue. 

Do you know any direct email or phone?

Thank you.

**Reply by amyegan** [reply]
It sounds like you already found a support email address. If you can share a case number from that, I can check the status and make sure it was sent to the right team.

**Reply by aserrajr** [reply]
@amyegan I have sent an email to registration@vercel.com with no answer so far. Do you have the correct email?

**Reply by amyegan** [reply]
That is the correct email address. Tickets are processed by the order in which they are received, so it can take some time for the team to respond to all of the earlier cases that were opened before they reach yours. 

For more information about response times, please visit https://vercel.com/guides/vercel-support-queue-time
